SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -85,6 +85,7 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	// Used by USpatialSpawner (when new players join the game) and USpatialInteropPipelineBlock (when player controllers are migrated).
 	USpatialNetConnection* AcceptNewPlayer(const FURL& InUrl, FUniqueNetIdRepl UniqueId, FName OnlinePlatformName, bool bExistingPlayer);
 
+	USpatialActorChannel* CreateActorChannel(AActor* Actor, UNetConnection* InConnection);
```

Comment:
I think we want to be clear that this is using a Spatial NetConnection and creates SpatialActorChannels.
```suggestion
	USpatialActorChannel* CreateSpatialActorChannel(AActor* Actor, USpatialNetConnection* InConnection);
```

Reply:
Applied the suggested change.